src/lib.rs

```diff
@@ -185,7 +185,7 @@ pub fn init_logging(opentelemetry_url: &Option<Url>) -> Result<(), LemmyError> {
     .trim_matches('"')
     .parse::<Targets>()?;
 
-  let format_layer = tracing_subscriber::fmt::layer().with_filter(targets.clone());
```

Comment:
Woodpecker failed because clippy was complaining about the use of clone. I made the suggested change and now woodpecker is failing because it won't compile. What do?